Updated Components Styling sections (#532)
IgniteUI/indigo-design-docfx: en/components/dropdown.md

## Styling

> The Dropdown comes with styling flexibility through an option controlling the background color, as well as various options related to the items it contains such as item and header background and text colors.

Reviewer: ```suggestion
The Dropdown comes with styling flexibility for its background color, as well as various options related to the elements it contains such as item and header background and text colors.
```
